utils/resultPrint.js

```diff
@@ -0,0 +1,36 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES, RESULT, RULES } from "../constants/messages.js";
+
+/**
+ * strike개수와 ball 개수 파악하여 값 출력 함수
+ * @param {strike와 ball 개수가 담긴 배열} result
+ * @returns
+ */
+const resultPrint = (result) => {
+  let strike = result[0];
+  let ball = result[1];
+
+  if (strike === 0 && ball === 0) {
+    MissionUtils.Console.print(RESULT.NOTHING);
+    return;
+  }
+  if (strike === RULES.NUM_LENGTH) {
+    MissionUtils.Console.print(RULES.NUM_LENGTH + RESULT.STRIKE);
+    MissionUtils.Console.print(GAME_MESSAGES.THREE_STRIKE);
+    return;
+  }
+
+  if (strike > 0 && ball === 0) {
+    MissionUtils.Console.print(`${strike}${RESULT.STRIKE}`);
+    return;
+  }
+
+  if (ball > 0 && strike === 0) {
+    MissionUtils.Console.print(`${ball}${RESULT.BALL}`);
+    return;
+  }
```

Comment:
```suggestion
  if (ball === 0) {
    MissionUtils.Console.print(strike + RESULT.STRIKE);
    return;
  }

  if (strike === 0) {
    MissionUtils.Console.print(ball + RESULT.BALL);
    return;
  }
```
위에서 둘 다 0인 경우를 제외 했기 때문에 조건문에서 `>` 조건은 제외해도 똑같이 동작할 것 같아요!

그리고 출력할 때 템플릿 리터럴 문법 사용해주셨는데 `+` 연산자를 사용해도 괜찮을 것 같아요~!

Reply:
아 헉 !! 그렇네요!! 
리팩토링때 꼭 고치는게 좋을 것  같아요👍